code_snippets/results/result.api-billing-usage-hosts.sh

```diff
@@ -0,0 +1,43 @@
+{
+  usage: [
+  {
+     "host_count": 7,
+     "gcp_host_names": "",
+     "apm_host_names": "davidm-test-host,i-00024a764a8c1fa15,i-0008643486a4b9911",
+     "container_count": 4,
+     "hour": "2017-07-24T00",
+     "apm_host_count": 8,
+     "agent_host_count": 9,
+     "apm_trace_count": 6,
+     "azure_host_names": "NetworkWatcherLinuxVM,WindowsVmV2,sqlserver",
+     "gcp_host_count": 0,
+     "azure_host_count": 2,
+     "aws_host_names": "i-00288fa84d5cd722b,i-00bb13e4e919977c2",
+     "agent_host_names": "davidm-test-host,i-00024a764a8c1fa15",
+     "aws_host_count": 2,
+     "unbillable_host_count": 0
+   },
```

Comment:
actually on that note, i think we decided not to expose names (or ids) at all, at least for now?   The high cardinality that can occur with larger orgs is not well-supported (e.g. thousands of names).  Another concern with host ids is we really shouldn't be exposing them internal ids to the public.